docs/dive-in/2024/02/2024-02-02-neon-db.md

```diff
@@ -0,0 +1,69 @@
+# Store data
+
+- Author: Magali Deslous-Paoli
+- Date : 2024-02-02
+
+## Summary
+
+Store our data in a shared DB
+
+## Context
+
+We will need to store our data into a shared database, accessible from our deployment. We do not need a lot of storage, but we do need simplicity and availability
```

Comment:
Can you specify deployment(s)?

Reply:
We need prod deployment at the very least. Added to the doc.